homeassistant/components/conversation/__init__.py

```diff
@@ -355,6 +355,21 @@ def async_get_agent_info(
     return None
 
 
+async def async_register_trigger_sentences(
```

Comment:
I think that we can remove this function if we are just exposing this functionality via a trigger